projects/plugins/jetpack/extensions/blocks/author-recommendation/author-recommendation.php

```diff
@@ -42,9 +58,26 @@ function load_assets( $attr, $content ) {
 	 */
 	Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
 
+	$wrapper_attributes = \WP_Block_Supports::get_instance()->apply_block_supports();
+	$placeholder_icon   = '<svg xmlns="http://www.w3.org/2000/svg" height="38px" viewBox="0 0 24 24" width="38px" fill="#646970"><path d="M12 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zm0 18l2-2 1-1v-2h-2v-1l-1-1H9v3l2 2v1.931C7.06 19.436 4 16.072 4 12l1 1h2v-2h2l3-3V6h-2L9 5v-.411a7.945 7.945 0 016 0V6l-1 1v2l1 1 3.13-3.13A7.983 7.983 0 0119.736 10H18l-2 2v2l1 1h2l.286.286C18.029 18.061 15.239 20 12 20z"/></svg>';
+
+	foreach ( $attr['recommendations'] as $recommendation ) {
+		$name      = empty( $recommendation['name'] ) ? '' : esc_html( $recommendation['name'] );
+		$url       = empty( $recommendation['URL'] ) ? '' : esc_attr( $recommendation['URL'] );
+		$site_icon = empty( $recommendation['site_icon'] ) ? '' : esc_attr( $recommendation['site_icon'] );
+
+		$icon_image = $site_icon ? "<img class='site-icon' src={$site_icon} alt={$name} />" : $placeholder_icon;
+
+		$content .= "<div class='recommendation-row'>
```

Comment:
Should we bail if we do not have enough information (empty name or URL) about a link?